start.sh

```diff
@@ -5,6 +5,7 @@ set -e
 
 cleanup () {
   docker compose -f ./docker-compose.dev.yml  down
+  docker stop oidc_client
```

Comment:
question: is it intentional to not remove the container with `docker rm`?

Reply:
The container will automatically get removed when it exits thanks to the `--rm` flag (see https://github.com/canonical/user-verification-service/blob/main/start.sh#L35)

Reply:
Nice thank you, I missed that!